recipes/r-lambertw/meta.yaml

```diff
@@ -0,0 +1,100 @@
+{% set version = '0.6.5' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-lambertw
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/LambertW_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/LambertW/LambertW_{{ version }}.tar.gz
+  sha256: 00614f6c5c07a6f22c4089926680b6b74676df24e6d04843556a90e87f68f9b0
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+  host:
+    - r-base
+    - r-mass
+    - r-rcolorbrewer
+    - r-rcpp >=1.0.4
+    - r-ggplot2
+    - r-lamw >=1.3.0
+    - r-reshape2
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-mass
+    - r-rcolorbrewer
+    - r-rcpp >=1.0.4
+    - r-ggplot2
+    - r-lamw >=1.3.0
+    - r-reshape2
+
+test:
+  commands:
+    - $R -e "library('LambertW')"           # [not win]
+    - "\"%R%\" -e \"library('LambertW')\""  # [win]
+
+about:
+  home: https://github.com/gmgeorg/LambertW http://arxiv.org/abs/0912.4554 http://arxiv.org/abs/1010.2265
+    http://arxiv.org/abs/1602.02200
+  license: GPL-2-or-later
```

Comment:
```suggestion
  license: GPL-2.0-or-later
```

Reply:
Ok thank you I will have to be more careful with the license